build-tooling.mk

```diff
@@ -0,0 +1,297 @@
+.DEFAULT_GOAL := help
```

Comment:
question: when we make changes in https://github.com/vmware-tanzu/build-tooling-for-integrations/tree/main/templates, or when user makes changes against this file in T-F by adding more components, what we can do to make things consistent ?

Reply:
it's up to the consumer(t-f) on when they want to consume the new release from https://github.com/vmware-tanzu/build-tooling-for-integrations, but when they update the Makefile in t-f it is their responsibility to update the components var..